### ISSUE TITLE
iojs 1.4.3, npm install build failed on node-gyp

### ISSUE THREAD

**Issue by yujinlim** [sass/node-sass#740]
System:
OS: osx 10.10.2
iojs: 1.4.3
npm: 2.6.1

```bash
> node-sass@2.0.1 install /Users/yujin/Documents/projects/<project_name>/main/node_modules/node-sass
> node scripts/install.js

Can not download file from https://raw.githubusercontent.com/sass/node-sass-binaries/v2.0.1/darwin-x64-iojs-1.4/binding.node

> node-sass@2.0.1 postinstall /Users/yujin/Documents/projects/<project_name>/main/node_modules/node-sass
> node scripts/build.js

module.js:322
    throw err;
          ^
Error: Cannot find module '/Users/yujin/Documents/projects/<project_name>/main/node_modules/node-sass/node_modules/pangyp/bin/node-gyp'
    at Function.Module._resolveFilename (module.js:320:15)
    at Function.Module._load (module.js:262:25)
    at Function.Module.runMain (module.js:485:10)
    at startup (node.js:112:16)
    at node.js:863:3
Build failed
node-sass@2.0.1 node_modules/node-sass
├── get-stdin@4.0.1
├── object-assign@2.0.0
├── replace-ext@0.0.1
├── nan@1.7.0
├── semver@4.3.1
├── shelljs@0.3.0
├── cross-spawn@0.2.6 (lru-cache@2.5.0)
├── mkdirp@0.5.0 (minimist@0.0.8)
├── chalk@0.5.1 (ansi-styles@1.1.0, supports-color@0.2.0, has-ansi@0.1.0, strip-ansi@0.3.0)
├── meow@3.1.0 (minimist@1.1.0, camelcase-keys@1.0.0, indent-string@1.2.1)
├── npmconf@2.1.1 (uid-number@0.0.5, inherits@2.0.1, osenv@0.1.0, ini@1.3.3, once@1.3.1, config-chain@1.1.8, nopt@3.0.1)
├── gaze@0.5.1 (globule@0.1.0)
├── request@2.53.0 (caseless@0.9.0, json-stringify-safe@5.0.0, aws-sign2@0.5.0, forever-agent@0.5.2, form-data@0.2.0, stringstream@0.0.4, oauth-sign@0.6.0, tunnel-agent@0.4.0, isstream@0.1.2, qs@2.3.3, combined-stream@0.0.7, mime-types@2.0.9, http-signature@0.10.1, bl@0.9.4, tough-cookie@0.12.1, hawk@2.3.1)
└── sass-graph@1.0.3 (commander@2.6.0, lodash@2.4.1, glob@4.5.0)
```

**Comment by xzyfer** [comment]
Thanks for the report @yujinlim but iojs > 1.2 isn't supported at the moment. This will be addressed in the soon to be released 3.0.0. You can downgrade to 1.2 or use nvm to manage multiple versions.

In the future please search for a similar issue before opening a new one.

**Comment by yujinlim** [comment]
@xzyfer thanks for the information!
